src/trace-plugin-interface.js

```diff
@@ -224,12 +225,14 @@ PluginAPI.prototype.wrap = function(fn) {
  * This is necessary in order to create child spans correctly in event handlers.
  * @param {EventEmitter} emitter An event emitter whose handlers should have
  * the trace context binded to them.
+ * @returns {EventEmitter} - returns the given event emitter
  */
 PluginAPI.prototype.wrapEmitter = function(emitter) {
   if (!this.agent_.namespace) {
     this.logger_.warn('Trace agent: No CLS namespace to bind emitter to');
   }
   this.agent_.namespace.bindEmitter(emitter);
+  return emitter;
```

Comment:
I would want this to be added once we reach a consensus that it would improve the interface, so please hold off on this change for now.